### PR TITLE
Fix #1101 - Status Bar is now changing as expected

### LINKED ISSUE #1101: Status Bar is not changing to Translucent Status Bar.
**Actual Behaviour**

The status bar is staying the same when I switch it to Translucent Status Bar.

**Expected Behaviour**

Status bar should be changed accordingly.

**Steps to reproduce it**

Open the app -> From overflow menu go to settings -> Under theme section choose 'Translucent status bar'. (Status bar will remain the same as it was earlier.)

**LogCat for the issue**

.

**Screenshots of the issue**
giphy

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by aditya98ak**
Fix # 1101

Changes: 
The status bar is now changing perfectly ( i,e, from normal to translucent, and vice-versa) but for perfect results, the app should be restated! I am notifying the end-user by showing up a Toast.

Screenshots for the change: 
.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java [hunk 1]
```diff
@@ -243,6 +244,7 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                 SP.putBoolean(getString(R.string.preference_translucent_status_bar), isChecked);
                 updateTheme();
                 setStatusBarColor();
+                Toast.makeText(SettingsActivity.this, "Please restart the app for better results!", Toast.LENGTH_SHORT).show();
```
Comment (mohitmanuja) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java]: Please add this string in string.xml

Comment (aditya98ak) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java]: okay, so I close this PR, and reopen a new one? RIght?

**Comment by mohitmanuja** [comment]
No, you can do changes in this PR. 
Follow this steps

1. Just do changes 
2. Git add 
3. Commit amend
4. Force push

**Comment by aditya98ak** [comment]
@mohitmanuja I did the changes, Please review :)

File: hs_err_pid18590.log [hunk 2]
```diff
@@ -0,0 +1,1295 @@
+#
```
Comment (mohitmanuja) [inline hs_err_pid18590.log]: Please remove this error file.

**Comment by aditya98ak** [comment]
@mohitmanuja Changes done!

**Comment by aditya98ak** [comment]
@mohitmanuja  Please review

File: hs_err_pid18590.log (hunk 2 above)
Comment (aditya98ak) [reply hs_err_pid18590.log]: @mohitmanuja Error file removed

**Pull request merged**